CMakeLists.txt

```diff
@@ -337,6 +337,10 @@ find_package(Boost ${VIAMCPPSDK_BOOST_VERSION_MINIMUM} REQUIRED COMPONENTS heade
 
 find_package(gRPC ${VIAMCPPSDK_GRPC_VERSION_MINIMUM} CONFIG)
 if (gRPC_FOUND)
+  if (NOT Protobuf_VERSION)
+    # Handle the situation where find_package(gRPC) does not populate Protobuf vars
+    find_package(Protobuf CONFIG)
```

Comment:
- I think the new call to `find_package` should be `REQUIRED`, since if the `find_package` for gRPC did not set these things, and then the `find_package` for `Protobuf` fails, thigns aren't going to work.
- Should `VIAMCPPSDK_PROTOBUF_VERSION_MINIMUM` be passed here too?

Reply:
I'd been thinking that as well--semantically proto is a directly-consumed SDK dependency so we ought to be `find_package`ing it directly rather than relying on gRPC to provide it to us.

The version minimum yes I think so, pending the resolution of what we discussed with there being a disagreement in which of the nonsensical versioning schemes is being used.